quickwit-indexing/src/split_store/indexing_split_store.rs

```diff
@@ -197,26 +197,29 @@ impl IndexingSplitStore {
     /// The output_path is expected to be a directory path.
     pub async fn fetch_split(
         &self,
-        split_id: &str,
+        split: &SplitMetadata,
```

Comment:
I don't see any other use than taking the split_id.

Reply:
Mmmm, sorry for this noise :/
Seems I don't know why I did that...